sycl/test-e2e/AtomicRef/atomic_memory_order_seq_cst.cpp

```diff
@@ -5,13 +5,23 @@
 #include <cmath>
 #include <iostream>
 #include <numeric>
+#include <sycl/aspects.hpp>
 using namespace sycl;
 
 constexpr size_t N_items = 128;
 
 size_t CalculateIterations(device &device, size_t iter_cap) {
-  uint64_t max_chars_alloc =
-      device.get_info<info::device::max_mem_alloc_size>() / sizeof(char);
+  uint64_t max_alloc_size = device.get_info<info::device::max_mem_alloc_size>();
+  // If querying free memory is supported, use that as the max for allocation.
+  if (device.has(aspect::ext_intel_free_memory)) {
+    namespace intel = sycl::ext::intel;
```

Comment:
Nit: Seems a little overkill to declare a namespace alias for one use.

Reply:
Yeah, thought it made the variable assignment here a readable one-liner this way, but it is overkill I guess. I'll push  the updates, cheers!

Reply:
Done.